internal/npm_install/generate_build_file.ts

```diff
@@ -76,42 +75,58 @@ package(default_visibility = ["${visibility}"])
 }
 
 if (require.main === module) {
-  main();
+  main()
+}
+
+
+async function exists(p: string) {
+  return await fs.access(p, constants.F_OK | constants.W_OK).then(() => true, () => false);
```

Comment:
Why is the `catch` necessary unlike previously in the sync version?

Reply:
[fs.exists](https://nodejs.org/dist/latest-v14.x/docs/api/fs.html#fs_fs_exists_path_callback) has been deprecated, so I use [fs. access](https://nodejs.org/dist/latest-v14.x/docs/api/fs.html#fs_fs_access_path_mode_callback) instead, `fs.acess` throws an error when the file doesn't match the   condition (visible and readable)